queries/special.py

```diff
@@ -462,22 +464,29 @@ def _play_film(qs: str, q: Query) -> AnswerType:
 
 _THREATS: AnswerType = {"answer": "Ei skal höggva!"}
```

Comment:
"Eigi skal höggva"?

Reply:
Hahaha, don't even remember adding that one. But yes, "eigi".